besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2572,20 +2572,8 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
     final EthNetworkConfig.Builder builder =
         new EthNetworkConfig.Builder(EthNetworkConfig.getNetworkConfig(network));
 
-    // custom genesis file use comes with specific default values for the genesis
-    // file itself
-    // but also for the network id and the bootnodes list.
     if (genesisFile != null) {
-
-      // noinspection ConstantConditions network is not always null but injected by
```

Comment:
is this noinspection comment actually read by errorprone?

Reply:
I don't think so - my IDE doesn't flag the following line after removing it

Reply:
ok